lerna.json

```diff
@@ -0,0 +1,4 @@
+{
+  "lerna": "2.0.0-beta.7",
+  "version": "1.1.35"
```

Comment:
@stephenkao when we bump to `2.0.0` with `lerna`. it will only bump the version of the changed packages. Seems like it might be better to bump all of the packages to `2.0.0` but we will have to do that manually 😢